nameable_decorator/decorators.rb

```diff
@@ -10,6 +10,6 @@ def correct_name
 class TrimmerDecorator < BaseDecorator
   def correct_name
     super()
-    @nameable.correct_name[0, 9]
+    @nameable.correct_name[0, 10]
```

Comment:
- [ ] The above condition can be expressed as:

```ruby
def correct_name
      @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
end
```

Reply:
Thanks for the suggestions @Cathella 😊